File: Common/InMotionGIT.Common - Copy/BackOffice/Values.vb
Namespace BackOffice

    Public Class Values

        Public Function insGetSetting(ByVal Name As String, ByVal DefValue As String, Optional ByVal Group As String = "") As String
            Dim lclsConfig As New VisualTimeConfig

            insGetSetting = lclsConfig.LoadSetting(Name, DefValue, Group)
            lclsConfig = Nothing

        End Function

    End Class

End Namespace

File: Common/InMotionGIT.Common/BackOffice/Values.cs

namespace InMotionGIT.Common.BackOffice
{

    public class Values
    {

        public string insGetSetting(string Name, string DefValue, string Group = "")
        {
            string insGetSettingRet = default;
            var lclsConfig = new VisualTimeConfig();

            insGetSettingRet = lclsConfig.LoadSetting(Name, DefValue, Group);
            lclsConfig = null;
            return insGetSettingRet;

        }

    }

}